AdyenCard/Validators/CardAddressValidator.swift

```diff
@@ -0,0 +1,60 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import Foundation
+
+/// Validates a card's address field.
+public final class CardAddressValidator {
+
+    private let configuration: CardComponent.Configuration
```

Comment:
This class does not seem to need the whole card configuration, only the address mode. It's better not to depend on the card configuration if we want to make this a generic address validator.

Reply:
Yes, You are right. I will change this.